index.html

```diff
@@ -148,48 +165,70 @@ <h2 class="proj-title">Multi-Post Stories</h2>
-          <li class="lang-item d-flex font500"><img src="icon/css.png" class="pad10" alt="CSS"/><span class="pad10">CSS</span></li>
-          <li class="about-item d-flex border font500">Frameworks<img src="icon/chevron-right.png" alt="chevron right"/></li>
-          <li class="about-item d-flex border font500">Skills<img src="icon/chevron-right.png" alt="chevron right"/></li>
+          <li class="about-item d-flex font500" tabindex="0">Languages
+            <img src="icon/chevron-down.png" alt="chevron down"/></li>
+          <li class="lang-item d-flex font500" tabindex="0">
+            <img src="icon/javascript.png" class="pad10" alt="JavaScript"/>
+            <span class="pad10">JavaScript</span></li>
+          <li class="lang-item d-flex font500" tabindex="0">
+            <img src="icon/html.png" class="pad10" alt="HTML"/>
+            <span class="pad10">HTML</span></li>
+          <li class="lang-item d-flex font500" tabindex="0">
+            <img src="icon/css.png" class="pad10" alt="CSS"/>
+            <span class="pad10">CSS</span></li>
+          <li class="about-item d-flex border font500" tabindex="0">Frameworks<img
+              src="icon/chevron-right.png" alt="chevron right" /></li>
+          <li class="about-item d-flex border font500" tabindex="0">Skills<img
+              src="icon/chevron-right.png" alt="chevron right" /></li>
         </ul>
       </div>
     </section>
     <footer class="contact d-flex">
-      <h2 class="white bold font40-52">Contact me</h2>
-      <p class="white font20-28">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
-      <form action="https://formspree.io/f/xdoypzpo" method="POST" class="form d-flex">
-        <input class="usr_detail smooth_border" type="text" name="usr-name" required maxlength="30" placeholder="Your name">
-        <input class="usr_detail smooth_border" type="email" name="usr-email" required placeholder="name@mail.com">
-        <textarea class="usr_comment smooth_border" name="usr-message" required maxlength="500" placeholder="Write your message here"></textarea>
+      <h2 class="white bold font40-52" tabindex="0">Contact me</h2>
+      <p class="white font20-28 bold" tabindex="0">If you have an application you are
+        interested in developing, a feature that you need built or a project
+        that needs coding. I’d love to help with it</p>
+      <form action="https://formspree.io/f/xdoypzpo" method="POST" class="form
+        d-flex">
+        <input class="usr_detail smooth_border" type="text" name="usr-name"
+          required maxlength="30" placeholder="Your name">
+        <input class="usr_detail smooth_border" type="email" name="usr-email"
+          required placeholder="name@mail.com">
+        <textarea class="usr_comment smooth_border" name="usr-message" required
+          maxlength="500" placeholder="Write your message here"></textarea>
```

Comment:
- [ ] Your form interacts well but these elements do not have a corresponding label. If a form control does not have a properly associated text label, the function or purpose of that form control may not be presented to screen reader users. Form labels also provide visible descriptions and larger clickable targets for form controls. Since there is no visible label from the template, you can achieve this by using the `aria-label` attribute. [See docs on how to use this WAI-ARIA property](https://www.w3.org/TR/wai-aria-1.1/#aria-label)